src/assert.js

```diff
@@ -98,122 +97,131 @@ Assert.prototype = {
 		}
 
 		return assert.test.pushResult( resultInfo );
-	},
+	}
+
+	ok( result, message ) {
+		if ( !message ) {
+			message = result ?
+				"okay" :
+				`failed, expected argument to be truthy, was: ${dump.parse( result )}`;
+		}
```

Comment:
@leobalter Not sure that will work. A custom message (two arguments passed) will always be clobbered by that code when the assertion value is falsy. Think the if condition should check for undefined.